firebase-crashlytics-ndk/src/main/jni/crashpad/crashpad_util/include/execvpe.h

```diff
@@ -0,0 +1,9 @@
+#ifndef CRASHPAD_UTIL_EXECVPE_H
+#define CRASHPAD_UTIL_EXECVPE_H
+
+#if __ANDROID_API__ < 21
+extern int
+execvpe(const char *__file, char *const *__argv, char *const *__envp);
```

Comment:
this should have an extern "C" around it

Reply:
Done